NullPointerException while exporting things via REST interface

<!-- Provide a general summary of the issue in the *Title* above -->

I am using the docker snapshot as of 09.04.2025 (should respond to the git commit of the same date)

Going to the developer tools, I am able to export "items" to the Clipboard.

Using the "things" link causes an Exception, with the following stack trace:

`
2025-04-09 19:21:31.043 [ERROR] [internal.JSONResponseExceptionMapper] - Unexpected exception occurred while processing REST request.
java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.length()" because "this.text" is null
        at java.util.regex.Matcher.getTextLength(Unknown Source) ~[?:?]
        at java.util.regex.Matcher.reset(Unknown Source) ~[?:?]
        at java.util.regex.Matcher.<init>(Unknown Source) ~[?:?]
        at java.util.regex.Pattern.matcher(Unknown Source) ~[?:?]
        at java.util.regex.Pattern.split(Unknown Source) ~[?:?]
        at java.util.regex.Pattern.split(Unknown Source) ~[?:?]
        at org.openhab.core.config.core.ConfigUtil.getDefaultValueAsCorrectType(ConfigUtil.java:71) ~[?:?]
        at org.openhab.core.thing.fileconverter.AbstractThingFileGenerator.getConfigurationParameters(AbstractThingFileGenerator.java:123) ~[?:?]
        at org.openhab.core.thing.fileconverter.AbstractThingFileGenerator.getConfigurationParameters(AbstractThingFileGenerator.java:92) ~[?:?]
        at org.openhab.core.model.thing.internal.fileconverter.DslThingFileConverter.buildModelThing(DslThingFileConverter.java:129) ~[?:?]
        at org.openhab.core.model.thing.internal.fileconverter.DslThingFileConverter.buildModelThing(DslThingFileConverter.java:141) ~[?:?]
        at org.openhab.core.model.thing.internal.fileconverter.DslThingFileConverter.generateFileFormat(DslThingFileConverter.java:86) ~[?:?]
        at org.openhab.core.io.rest.core.internal.fileformat.FileFormatResource.createFileFormatForAllThings(FileFormatResource.java:234) ~[?:?]

`

<!-- Important: Please contact the openHAB community forum for questions or -->
<!-- for configuration and usage guidance: https://community.openhab.org -->

<!-- Feel free to delete any comment lines in the template (starting with "<!--") -->

## Expected Behavior
<!-- If you're describing a bug, tell us what should happen -->
<!-- If you're suggesting a change/improvement, tell us how it should work -->

Exporting things should work :)

## Current Behavior
<!-- If describing a bug, tell us what happens instead of the expected behavior -->
<!-- Include related log information (preferably debug level) and related configs -->
<!-- Use a file attachment for log and config information longer than a few lines -->
<!-- Enclose multi-line log/code snippets with ``` on new lines for proper formatting -->
<!-- If suggesting a change/improvement, explain the difference from current behavior -->
<!-- For improvements, discuss at community.openhab.org first and include link to topic -->

## Possible Solution
<!-- Not obligatory, but suggest a fix/reason for the bug, -->
<!-- or ideas how to implement the addition or change -->

## Steps to Reproduce (for Bugs)
<!-- Provide a link to a live example, or an unambiguous set of steps to -->
<!-- reproduce this bug. Include code to reproduce, if relevant -->
1.
2.

## Context
<!-- How has this issue affected you? What are you trying to accomplish? -->
<!-- Providing context helps us come up with a solution that is most useful in the real world -->

The last thing I did beforehand was creating two "hue" bridges via IP address entry using the admin UI.

Removing those and retrying the "export" click did not help either. I have no idea which "thing" might cause this issue.

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Version used: (e.g., openHAB and add-on versions)
* Environment name and version (e.g. Chrome 111, Java 17, Node.js 18.15, ...):
* Operating System and version (desktop or mobile, Windows 11, Raspbian Bullseye, ...):

* openHAB 5.0.0-snapshot, dockerized, running on a Linux host.
* Browser n/a, fails with curl the same way

--- a/bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/ConfigUtil.java
+++ b/bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/ConfigUtil.java
@@ -68,6 +68,9 @@ public class ConfigUtil {
      */
     public static @Nullable Object getDefaultValueAsCorrectType(ConfigDescriptionParameter parameter) {
         if (parameter.isMultiple()) {
+            if (parameter.getDefault() == null) {
+                return null;
+            }
             List<Object> defaultValues = Stream.of(DEFAULT_LIST_SPLITTER.split(parameter.getDefault())) //
                     .map(value -> value.trim().replace("\\,", ",")) //
                     .filter(not(String::isEmpty)) //

